src/ui/components/Notice/styles.scss

```diff
@@ -11,6 +11,7 @@
   grid-gap: 4px;
   grid-template-columns: min-content auto;
   line-height: 1.1;
+  margin-bottom: 10px;
```

Comment:
This will probably add a `margin` to all `Notice` components, which might not be a good idea.

Reply:
Yes, you are correct, it will add margin to every Notice component.
 Should I define inline css for it?

Reply:
I think here https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/Addon/styles.scss#L67 ;)

Reply:
Then, I'll add a class name to the `Notice` you've added and change the SCSS like this:

```scss
.AddonCompatibilityError,
.YourNewClassName {
  ...
}
```

Reply:
Got it :dancer:

Reply:
Fixed  :man_dancing: 

After changes
![screenshot_2018-08-24 idkhtml add-ons for firefox 1](https://user-images.githubusercontent.com/22130317/44589013-f7607800-a7d4-11e8-94a6-6094ee86c5e5.png)
![screenshot_2018-08-24 idkhtml add-ons for firefox](https://user-images.githubusercontent.com/22130317/44589014-f7607800-a7d4-11e8-939a-0177f930e3ad.png)